WooCommerce/Classes/ViewRelated/Orders/Refund/RefundSubmissionUseCase.swift

```diff
@@ -291,15 +291,21 @@ private extension RefundSubmissionUseCase {
         DDLogError("Failed to refund: \(error.localizedDescription)")
         // Informs about the error.
         //TODO: Check which refund errors can't be retried, and use that to call nonRetryableError(from: self.rootViewController, error: cancelError)
```

Comment:
nit: maybe we can remove this TODO in this PR?

Reply:
Yeah, removed ✅